docker/ci/DOCKERHUB_DESCRIPTION.md

```diff
@@ -0,0 +1,69 @@
+# Eclipse GLSP CI
```

Comment:
Is there a reason this file is called DOCKERHUB_DESCRIPTION and not simply README since it is in it's own directory anyway?

Reply:
I was planing on using a sync-tool that required that naming scheme. However, since we don't have the necessary rights anyways and description updates have to be handled manually by the Eclipse Webmaster we can also simply call this README